src/main.cpp

```diff
@@ -28,6 +29,12 @@ int main(int argc, char *argv[])
    QFontDatabase::addApplicationFont(":/DejaVuSans");
    QFontDatabase::addApplicationFont(":/DejaVuSansMono");
 
+   QTranslator qtTranslator;
+   if (qtTranslator.load(QLocale(), "gitqlient", "_", ":/translations/"))
```

Comment:
I think that in newer versions of Qt, it will complain if we don't wrap raw strings in QString::fromUtf8 or any other method:
```suggestion
   if (qtTranslator.load(QLocale(), QString::fromUtf8("gitqlient"), QString::fromUtf8("_"), QString::fromUtf8(":/translations/")))
```